app/models/restaurant.rb

```diff
@@ -1,11 +1,12 @@
 class Restaurant < ActiveRecord::Base
-  serialize :hours   ### gets a more complicated data structure to be able to be inputed into the database
+ 
+  serialize :hours   
```

Comment:
no longer necessary - the hours column should be dropped from the db once the data in it has been moved to the daily_schedules table